static/js/public/first-snap-flow.js

```diff
@@ -168,19 +168,20 @@ function push() {
   }
 
   getCount(snapName => {
+    // Enable "Continue" button
     const continueBtn = document.querySelector(".js-continue");
     if (continueBtn) {
       continueBtn.href = `/${snapName}/listing?from=first-snap`;
```

Comment:
I should have written in the issue, not just said at standup, could this link to the release UI?
```suggestion
      continueBtn.href = `/${snapName}/releases?from=first-snap`;
```

Reply:
Ahhh...sorry. Will update.